spec/requests/orders_request_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe OrdersController, type: :controller do
+  let(:admin) { FactoryBot.create(:user) }
+  let(:order) { FactoryBot.create(:order) }
+
+  before(:each) do
+    sign_in admin
+  end
+
+  describe 'GET #index' do
+    it 'returns all orders' do
+      get :index
+
+      expect(response).to have_http_status(200)
+    end
+
+    it 'returns all orders' do
+      get :index,  params: { status: 'Pending' }
+
+      expect(response).to have_http_status(200)
```

Comment:
expect for the returned records that `status.uniq.length` is only 1 and it is Pending.